ui/src/app/pages/data/concept/concept-homepage.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import {mount} from 'enzyme';
```

Comment:
I was hoping to see a test case that verifies the fix here and would protect against regressions. Is there a reason it would be infeasible or prohibitively expensive  to write such a test?

Reply:
Seems like a good idea.